CMakeLists.txt

```diff
@@ -1,5 +1,7 @@
 cmake_minimum_required(VERSION 3.15)
 
+set(CMAKE_OSX_DEPLOYMENT_TARGET "10.15" CACHE STRING "Minimum OS X deployment version" FORCE)
```

Comment:
Would it make sense to set this in the `ci/build-universal` script instead?

Reply:
I set it for all builds so its easier for devs to figure out why CI is breaking.

If you feel I should move it into the script let me know.

Reply:
I'm OK with leaving it, but I still don't understand why that would make it easier for devs (who are they?) to figure out why CI builds fail.